Code/Framework/AzFramework/AzFramework/Application/Application.cpp

```diff
@@ -227,6 +228,14 @@ namespace AzFramework
                 using AssetCatalogBus = AZ::Data::AssetCatalogRequestBus;
                 AssetCatalogBus::Broadcast(AZStd::move(StartMonitoringAssetsAndLoadCatalog));
             }
+#if !defined(_RELEASE)
```

Comment:
In your IRemoteTools.h possibly, or even as a PAL trait, I would make a new define that guards all the IRemoteTools touchpoints for a couple reasons:
1 - if someone decides they really need to turn it on in release it's a 1-line change
2 - if someone wants to search and find all the touchpoints, there's a unique define that clearly tags all touchpoints

Something like
```
#if !defined(_RELEASE)
#   define ENABLE_REMOTE_TOOLS 1
#endif
```